### ISSUE TITLE
Publish Storybook to GitHub Pages

### ISSUE THREAD

**Issue by sareh** [BBC-archive/psammead#37]
**EDIT (~Chris): publish to GitHub Pages, update README & Git repo to link to live Storybook, and add instructions to Definition of Done so that we remember to do this when we merge things.**

-------------

**Is your feature request related to a problem? Please describe.**
Currently, there is a manual command for building storybook in the Psammead repository's package.json https://github.com/BBC-News/psammead/blob/latest/package.json#L12

We should only run a `build-storybook` command on CI, since currently anyone can accidentally overwrite the main Storybook with a local version of their repository, rather than what's on `latest`.

**Describe the solution you'd like**
- We should remove the `simorgh:export` script from the package.json
- We should add the Storybook build step in a CI step.  
- This build step should only run on a merge into `latest`

This should be in Travis CI. Since this is an open source project and should be accessible by non-BBC employees, we shouldn't use Jenkins CI. 

We can refer to the example from the Simorgh repository: https://github.com/BBC-News/simorgh/blob/latest/.travis.yml

```
     - stage: deploy
      if: type = push AND branch = latest
      before_script: npm install now --no-save
      script:
        - build-storybook -c .storybook -o simorghstorybook
        - now --token $now_token ./simorghstorybook --public && now alias --token $now_token
        - now rm -y --token $now_token --safe simorghstorybook
      after_script: now rm -y --token $now_token --safe simorghstorybook
  allow_failures:
    - after_script: now rm -y --token $now_token --safe simorghstorybook
```
However, we should *not* be using Now.  

**Describe alternatives you've considered**
We could not make this change. However, it'd mean our storybook could easily go out of date. 

**Testing notes**
Testing that on a merge into `latest`, Travis CI runs and builds Storybook and makes it available at the correct endpoint.

**Additional context**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

**Comment by sareh** [comment]
Note, when picking up this issue, it would be useful to change the port number of storybook locally away from `8080`, to another port.  See here for the relevant issue https://github.com/BBC-News/psammead/issues/5 @dr3

**Comment by sareh** [comment]
This is dependent on this ticket, which is setting up Jenkins CI: https://github.com/BBC-News/psammead/issues/46

**Comment by ChrisBAshton** [comment]
#46 is done, so not blocked on that anymore.

But blocked pending decision on public-facing Psammead Storybook URL. Assigned to @BogdanDogaru for now.

**Comment by ChrisBAshton** [comment]
Option: explore publishing to GitHub pages.

**Comment by dr3** [comment]
Possible gh-pages option https://github.com/storybooks/storybook-deployer

Builds storybook and deploys it to a gh-pages branch

**Comment by ChrisBAshton** [comment]
Agreement in refinement:

* manual deployment to github pages for now (with instructions in 'Definition of Done' else we'll forget) - and add URL to Readme and GitHub repo root


We'll raise a new issue to automate on merge in CI pipeline
